### ISSUE TITLE
Fix for non looping background music in public 1.10.9

### ISSUE THREAD

**Issue by ED-E** [50DKP/FF2-Official#404]
I'm not firm how this here works, so I'm sorry if I open and issue even though it's a solution to an issue.

Section: **Timer_PrepareBGM**
Move the kill timer/handle cleaner before you send it to PLAYBGM, then the background music looping works again. 

Something like this below:
```sp
if(MusicTimer[client]!=INVALID_HANDLE)
{
					KillTimer(MusicTimer[client]);
					MusicTimer[client]=INVALID_HANDLE;
}
if(CheckRoundState()==1 && (!currentBGM[client][0] || !StrEqual(currentBGM[client], "ff2_stop_music", false)))
{
					PlayBGM(client);
}
```

**Comment by Deathreus** [comment]
404: Issue not found

This issue was already dealt with [here](https://github.com/50DKP/FF2-Official/pull/402#issuecomment-226200549)

**Comment by 50Wliu** [comment]
What @Deathreus said, in addition to the fact that 1.10.10 hasn't actually been released yet.

Will keep this open though as it hasn't been fixed yet.

**Comment by 50Wliu** [comment]
Should be fixed by #402.
